platform/darwin/test/MGLFilterTests.m

```diff
@@ -0,0 +1,69 @@
+#import "MGLStyleLayerTests.h"
+
+@interface MGLFilterTests : MGLStyleLayerTests {
+    MGLGeoJSONSource *source;
+    MGLLineStyleLayer *layer;
+}
+@end
+
+@implementation MGLFilterTests
+
+- (void)setUp
+{
+    [super setUp];
+    NSString *filePath = [[NSBundle bundleForClass:self.class] pathForResource:@"amsterdam" ofType:@"geojson"];
+    NSURL *url = [NSURL fileURLWithPath:filePath];
+    NSData *geoJSONData = [NSData dataWithContentsOfURL:url];
+    source = [[MGLGeoJSONSource alloc] initWithSourceIdentifier:@"test-source" geoJSONData:geoJSONData];
+    [self.mapView.style addSource:source];
+    layer = [[MGLLineStyleLayer alloc] initWithLayerIdentifier:@"test-layer" sourceIdentifier:@"test-source"];
+}
+
+- (void)tearDown
+{
+    [self.mapView.style removeLayer:layer];
+    [self.mapView.style removeSource:source];
+}
+
+- (NSArray<NSPredicate *> *)predicates
+{
+    NSPredicate *equalPredicate = [NSPredicate predicateWithFormat:@"type == 'neighborhood'"];
```

Comment:
Mapbox Streets v7 uses the British spelling, `neighbourhood`, for consistency with OpenStreetMap.